Anytype/Sources/PresentationLayer/Modules/Membership/InitialScreen/MembershipModuleView.swift

```diff
@@ -84,13 +86,13 @@ struct MembershipModuleView: View {
     var legal: some View {
         VStack(alignment: .leading) {
             MembershipLegalButton(text: Loc.Membership.Legal.details) {
-                openURL(URL(string: AboutApp.pricingLink)!)
+                safariUrl = URL(string: AboutApp.pricingLink)!
             }
             MembershipLegalButton(text: Loc.Membership.Legal.privacy) { 
-                openURL(URL(string: AboutApp.privacyPolicyLink)!)
+                safariUrl = URL(string: AboutApp.privacyPolicyLink)!
             }
             MembershipLegalButton(text: Loc.Membership.Legal.terms) { 
-                openURL(URL(string: AboutApp.termsLink)!)
+                safariUrl = URL(string: AboutApp.termsLink)!
             }
```

Comment:
Looks like we don't need force unwrapping here

Reply:
agree. potential crash